apps/web/app/api/bookmark/route.ts

```diff
@@ -0,0 +1,57 @@
+import prisma from "@repo/db/client";
+
+export async function POST(req: Request) {
+  const { userid, trackid } = await req.json();
```

Comment:
You should have POST, DELETE, GET, PUT here in  a single file
No need for 4 saparate files with apis